project/AkkaDisciplinePlugin.scala

```diff
@@ -39,9 +39,22 @@ object AkkaDisciplinePlugin extends AutoPlugin with ScalafixSupport {
       Compile / scalacOptions ++= disciplineScalacOptions,
       Compile / scalacOptions --= undisciplineScalacOptions,
       Compile / console / scalacOptions --= Seq("-deprecation", "-Xfatal-warnings", "-Xlint", "-Ywarn-unused:imports"),
+      // -Ywarn-unused:_ breaks 'sbt ++2.13.0-M5 akka-actor/doc'
+      // https://github.com/akka/akka/issues/26119
+      Compile / doc / scalacOptions ++= (CrossVersion.partialVersion(scalaVersion.value) match {
```

Comment:
Why not disable all diagnostic flags for Scaladocs in the first place?

Reply:
👍 I should have thought of that originally, thanks for catching @jrudolph and @raboof :)

Reply:
Changes the scope of this, perhaps we should edit the title of this issue to clarify this larger point?